elastic/logs/templates/component/auditbeat-mappings.json

```diff
@@ -1785,6 +1785,9 @@
             "message": {
               "norms": false,
               "type": "text"
+              {% if index_sorting %},
+              "store": true
+              {% endif %}
```

Comment:
Hopefully one day this will work automagically when synthetic source is enabled.

Reply:
Yes, we should just fix this. For the `match_only_text` this is working as expected. If synthetic source is enabled then store defaults to true.